jaxlib/lapack.pyx

```diff
@@ -147,13 +164,14 @@ cdef void blas_ztrsm(void* out, void** data) nogil:
   cdef int32_t diag = (<int32_t*>(data[3]))[0]
   cdef int m = (<int32_t*>(data[4]))[0]
   cdef int n = (<int32_t*>(data[5]))[0]
-  cdef double complex* alpha = <double complex*>(data[6])
-  cdef double complex* a = <double complex*>(data[7])
-  cdef double complex* b = <double complex*>(data[8])
+  cdef int n_b = (<int32_t*>(data[6]))[0]
```

Comment:
(Be consistent in names. Here `batch` as suggested above.)

Reply:
have changed.